### ISSUE TITLE
import visualdl 之后日志无输出

### ISSUE THREAD

**Issue by supercyt** [PaddlePaddle/VisualDL#938]
关键代码：
```
import visualdl

logging.basicConfig(level=logging.INFO, format='%(message)s')

# 无输出
logging.info('Epoch %d, Iteration %d, loss = %.4f, auc: %.2f' % ( epoch, t, train_loss, train_auc_val))
```

**Comment by rainyfly** [comment]
您好，问题可能不出在import visualdl上哦，您可以试着只跑这三行代码，可以发现是可以输出的。
import visualdl
import logging
logging.basicConfig(level=logging.INFO, format='%(message)s')
logging.info('Epoch %d, Iteration %d, loss = %.4f, auc: %.2f' % ( 1, 1, 0.8888, 88))
